app/util/report-aggregator.test.ts

```diff
@@ -78,7 +80,8 @@ const reports: CovidReport[] = [
       [Symptom.NO_TASTE]: false,
       [Symptom.NO_SMELL]: false,
       [Symptom.SLIME_COUGH]: false,
-      [Symptom.RUNNY_NOSE]: false
+      [Symptom.RUNNY_NOSE]: false,
```

Comment:
Can't read this without thinking of an actual nose running 👃 🏃

Reply:
😆